### PR TITLE
Config panel ynh arguments

### PR DESCRIPTION
**Pull request by Psycojoker**
Matching PR for https://github.com/YunoHost/yunohost/pull/734

### CODE REVIEW

File: src/js/yunohost/controllers/apps.js [hunk 1]
```diff
@@ -430,6 +435,7 @@
             if (args[k].type == 'boolean') {
                 args[k].inputType = 'checkbox';
 
+                console.log(args[k]);
```
Comment (decentral1se) [inline src/js/yunohost/controllers/apps.js]: Debugging only or?

File: src/views/app/app_config-panel.ms [hunk 2]
```diff
@@ -28,40 +29,51 @@
...
-                            <input type="radio" name="{{id}}" value="false"{{#if (eq value false)}}checked{{/if}}> no
-                            <span class="help-block">{{ help }}</span>
-                            {{/if}}
+
+                                {{#if choices}}
+                                <select id="{{name}}" name="{{name}}" required class="form-control" {{{attributes}}}>
+                                    {{#choices}}<option value="{{value}}" {{#if selected}}selected{{/if}}>{{label}}</option>{{/choices}}
+                                </select>
+                                {{else if (eq inputType "checkbox")}}
+                                <input type="{{inputType}}" id="{{name}}" name="{{name}}" class="form-control auto-width" value="{{default}}" placeholder="{{example}}" {{required}} {{{attributes}}}>
```
Comment (decentral1se) [inline src/views/app/app_config-panel.ms]: `{{{` on attributes is correct or? Not familiar with this syntax.

File: src/js/yunohost/controllers/apps.js (hunk 1 above)
Comment (Psycojoker) [reply src/js/yunohost/controllers/apps.js]: Yes, PRs aren't finished yet

File: src/views/app/app_config-panel.ms (hunk 2 above)
Comment (Psycojoker) [reply src/views/app/app_config-panel.ms]: Came from here from where the code is adapted from https://github.com/YunoHost/yunohost-admin/blob/stretch-unstable/src/views/app/app_install.ms#L68

Tbh I don't know either, I think it's simply the `**kwargs` of python in mustache templates.

File: src/js/yunohost/controllers/apps.js (hunk 1 above)
Comment (decentral1se) [reply src/js/yunohost/controllers/apps.js]: Ah right. You can add "WIP" to the title and Github will make this PR a "draft". I'll resolve comments.

**Pull request merged**